soc/nxp/imxrt/imxrt118x/Kconfig.defconfig

```diff
@@ -23,12 +23,15 @@ config DCDC_VALUE
 config FLEXSPI_CONFIG_BLOCK_OFFSET
 	default 0x400 if BOOT_FLEXSPI_NOR
 
+config ROM_START_OFFSET
+	default 0xb000 if BOOT_FLEXSPI_NOR && SOC_SERIES_IMXRT118X
```

Comment:
Isn't this already dependent on `SOC_SERIES_IMXRT118X`, since it is within the `if SOC_SERIES_IMXRT118X` block?

Reply:
yes, it was a copy paste, I'm not sure why you phrased it like a question, I'll just update it to remove it in case I need to update this PR again, otherwise not worth the CI rerun IMO